docs/approach.rst

```diff
@@ -233,17 +233,19 @@ For this PCA method in particular, ``--tedpca`` provides three different options
 to select the PCA components based on three widely-used model selection criteria:
 
 * ``mdl``: the Minimum Description Length (`MDL`_), which is the most aggressive option;
-  i.e. returns the least number of components. This option is the **default and recommeded**
-  as we have seen it yields the most reasonable results.
+  i.e. returns the least number of components.
 * ``kic``: the Kullback-Leibler Information Criterion (`KIC`_), which stands in the
   middle in terms of aggressiveness. You can see how KIC is related to AIC `here`_.
 * ``aic``: the Akaike Information Criterion (`AIC`_), which is the least aggressive option;
-  i.e., returns the largest number of components.
+  i.e., returns the largest number of components. We have chosen AIC as the default PCA
+  criterion because it tends to result in fewer components than the Kundu methods, which increases
+  the likelihood that the ICA step will successfully converge, but also, in our experience, retains
+  enough components for meaningful interpretation later on.
 
 .. note::
     Please, bear in mind that this is a data-driven dimensionality reduction approach. The default
-    option ``mdl`` might not yield perfect results on your data. We suggest you explore the ``kic``
-    and ``aic`` options if running ``tedana`` with ``mdl`` returns less components than expected.
+    option ``aic`` might not yield perfect results on your data. We suggest you explore the ``kic``
+    and ``mdl`` options if running ``tedana`` with ``aic`` returns more components than expected.
```

Comment:
```suggestion
    option ``aic`` might not yield perfect results on your data. Consider ``kic``
    and ``mdl`` options if running ``tedana`` with ``aic`` returns more components than expected. There is no definitively right number of components, but, for typical fMRI datasets, if the PCA explains more than 98% of the variance or if the number of components is more than half the number of time points, then it may be worth considering more aggressive threshold.
```

I just tested AIC on some 5-echo data with 160 volumes and I got 99.28% variance explained and 134 components. Given the recent spate people with too few components, I think it's better to default to giving too many rather than too few, but I wanted to expand this comment a bit.

Reply:
Sounds good to me!